lib/jxl/modular/encoding/context_predict.h

```diff
@@ -282,15 +288,15 @@ struct FlatDecisionNode {
     PropertyVal splitval0;
     Predictor predictor;
   };
-  uint32_t childID;  // childID is ctx id if leaf.
   // Property+splitval of the two child nodes.
   union {
     PropertyVal splitvals[2];
     int32_t multiplier;
   };
+  uint32_t childID;  // childID is ctx id if leaf.
   union {
-    int32_t properties[2];
-    int64_t predictor_offset;
+    int16_t properties[2];
+    int32_t predictor_offset;
```

Comment:
Spec says this is read as `leaf_node.offset = UnpackSigned(DecodeHybridVarLenUint(3));`

So I think that implies it fits in `int32_t` — I think we define hybrid uint results to fit in `uint32_t`, and UnpackSigned should turn that into something that fits in `int32_t`. In any case, an offset that doesn't fit in `int32_t` doesn't make any sense since the values themselves are supposed to fit in `int32_t` (so a residual of zero would be invalid if the offset is so large).